Dockerfile

```diff
@@ -0,0 +1,61 @@
+# translate/pootle
+#
+# VERSION	0.0.1
+
+FROM debian
+
+MAINTAINER Ryan Northey <ryan@synca.io>
+
+RUN apt-get update && \
+    apt-get install -y \
+            python-pip \
+	    libxml2-dev \
+	    libxslt-dev \
+	    python-dev \
+	    zlib1g-dev \
+	    build-essential \
+	    redis-server \
+	    sudo && \
+    apt-get clean && \
+    pip install virtualenv && \
+    groupadd -r pootle && \
+    useradd -m -d /home/pootle \
+            -k /etc/skel \
+            -s /bin/bash \
+            -g pootle pootle && \
+    /etc/init.d/redis-server start && \
+    sudo -u pootle bash -c "\
+    	 mkdir ~/dev/env -p && \
+	 cd ~/dev/env && \
+    	 virtualenv . && \
+    	 . bin/activate && \
```

Comment:
I don't think it such a big deal. Another reason to use it is that we tell to use it in the docs, so the final result is closer to what we are used to, in case we need to provide support.